src/targets/gpu/multinomial.cpp

```diff
@@ -0,0 +1,33 @@
+#include <migraphx/gpu/multinomial.hpp>
+#include <migraphx/gpu/device/multinomial.hpp>
+#include <migraphx/gpu/context.hpp>
+#include <migraphx/tune_axis.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace gpu {
+
+shape hip_multinomial::compute_shape(const std::vector<shape>& inputs) const
+{
+    check_shapes{inputs, *this}.has(3).only_dims(2);
```

Comment:
Can we call the op.compute_shape() for the output shape?

Reply:
Fixed